client/src/components/menubar/style.ts

```diff
@@ -0,0 +1,74 @@
+import styled from "@emotion/styled";
+import { spacesS } from "../theme";
+import { getFeatureFlag } from "../../util/featureFlags/featureFlags";
+import { FEATURES } from "../../util/featureFlags/features";
+
+export const fullVerticalScreenWidth = "1275px";
+const isTest = getFeatureFlag(FEATURES.TEST);
+const isDownload = getFeatureFlag(FEATURES.DOWNLOAD);
+
+let firstBreakpoint = "1400px";
+if (isTest && isDownload) {
+  firstBreakpoint = "1455px";
+} else if (isTest || isDownload) {
+  firstBreakpoint = "1430px";
+}
+
+export const MenuBarWrapper = styled.div`
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  width: 100%;
+  position: relative;
+`;
+
+export const ResponsiveMenuGroupOne = styled.div`
+  display: flex;
+  flex-direction: row-reverse;
+  flex-wrap: wrap;
+  justify-content: right;
+  @media screen and (max-width: ${fullVerticalScreenWidth}) {
+    flex-direction: column-reverse;
+    position: absolute;
+    top: 37px;
+  }
+`;
+
+export const ResponsiveMenuGroupTwo = styled.div`
```

Comment:
discussion: What's your thought on using container query for this instead of viewport width 😄 ?

Feels like we might be able to use the menubar as the parent container and its width as a condition to know when each responsive group should resize?

https://css-tricks.com/css-container-queries/

Example: https://github.com/emotion-js/emotion/issues/2876#issuecomment-1345302543

```tsx
import "./styles.css";
import styled from "@emotion/styled";

const HotPinkContainer = styled.div`
  container-type: inline-size;
`;

const StyledHotPink = styled.div`
  padding: 8px;
  border: 1px solid #000;
  @container (min-width: 300px) {
    background-color: hotpink;
  }
`;

const HotPink = ({ children }) => {
  return (
    <HotPinkContainer>
      <StyledHotPink>{children}</StyledHotPink>
    </HotPinkContainer>
  );
};

export default function App() {
  return (
    <div className="App">
      <HotPink>
        kjfashdlfashdölfjasdölkfjasöldkjfaölsdkhfölasdjfölafjdsöl
      </HotPink>
    </div>
  );
}
```

Reply:
Love the resize observer! Not hogging JS thread is always helpful 🎉 Brilliant 👏